File: php/delete_credentials.php
#!/usr/bin/env php
<?php
require '_client.php';
$trunk = getenv('TRUNK_SID');
$cid = getenv('CREDENTIAL_ID');
if (!$trunk || !$cid) {
    echo "Error: TRUNK_SID and CREDENTIAL_ID required\n";
    exit(1);
}
echo "Deleting credential...\n";
exo_delete("/trunks/$trunk/credentials", ['id' => $cid]);

File: go/delete_credentials.go
package main

import (
	"fmt"
	"os"
)

func main() {
	trunk := os.Getenv("TRUNK_SID")
	cid := os.Getenv("CREDENTIAL_ID")
	if trunk == "" || cid == "" {
		fmt.Println("Error: TRUNK_SID and CREDENTIAL_ID required")
		os.Exit(1)
	}
	fmt.Println("Deleting credential...")
	deleteReq("/trunks/"+trunk+"/credentials", map[string]string{"id": cid})
}
